### ISSUE TITLE
[Task]: ODC for Ethnio

### ISSUE THREAD

**Issue by sumiat** [HHS/simpler-grants-gov#891]
Procurement for the design/research tool, Ethnio

**Comment by EOKENAVA** [comment]
Requested already. In progress

**Comment by lucasmbrown-usds** [comment]
@EOKENAVA @sarahknoppA6 -- why is this blocked right now?

**Comment by EOKENAVA** [comment]
@sarahknoppA6 - Prior to Sarah's vacation, she mentioned working with Julius and MH on this. Hopefully we can get some update from Sarah tomorrow

**Comment by sarahknoppA6** [comment]
@lucasmbrown-usds A6 and Nava had received notices last month from MicroHealth that they had been waiting on additional ODC funding for these licenses. I let Julius know before I left for vacation that we had a blocker on this, but we have not at this time received any update from MH nor Julius.

**Comment by EOKENAVA** [comment]
Waiting on feedback from MH on funding

**Comment by margaretspring** [comment]
@andycochran @rishalee  Based on Andy's comment in Slack that "consent forms, translation/localization, and API integration (e.g. importing contacts). And cap out at lower interaction amounts," would you re-assess the core features needed & what level licenses would provide them? I'm hoping this will identify if a more affordable level gets us most of what we need so we can go forward. Or determine the price point is prohibitive & figure out another direction. Thank you!

**Comment by andycochran** [comment]
We have confirmed a direction with Nava leadership for research tools, and have a recommendation for how to handle participant management and research incentives  disbursement to best accomplish program objectives. 

## Context
- ADR: [Use Ethnio for design research](https://github.com/HHS/simpler-grants-gov/blob/main/documentation/decisions/adr/2023-11-14-design-research-tools.md) (November 14, 2023) 
- Nava understood that MicroHealth would procure Ethnio licenses for managing engagements, that incentives funding would be loaded into the tool by HHS/MicroHealth, and that those funds would be available for Nava researchers to compensate research participants. 
- Unfortunately, MicroHealth was quoted a cost-prohibitive $50K+ for 8 logins billed annually (April 3, 2024)
- This rate for an "Enterprise" license (required for participant management feature) being cost-prohibitive, Nava has been working hard to determine and recommend alternative tooling and processes.

## Additional notes on incentives disbursement: 
- HHS noted that their policies forbid gift cards without a waiver/exception. Nava has been able to get clearance to disburse gift cards for State & Federal government customers, and can provide the justification HHS needs to get clearance.
- Direct monetary options (Venmo, Paypal, and ACH) — if allowed — are more difficult for reporting. Nava recommends only disbursing incentives as gift cards via Tremendous. It is auditable, defensible, and provides everything needed to substantiate the expense and obtain disbursement receipts. 

# Proposed solution
**Separate the concerns of participant management and incentives disbursement**

### Participant management
- Nava will absorb the cost of Ethnio, using their limited access licenses for participant management, scheduling, screeners, intercepts, etc. 
- We do not use Ethnio's incentives disbursement feature. 

### Incentives
- MicroHealth/HHS will set up a [Tremendous](https://www.tremendous.com/) account for incentives disbursement.
- Tremendous accounts are free. However, there's an additional charge for ACH, PayPal, and Venmo (no fees for other options). Nava can share its previous experience, but the determination of the dispersal method will be determined by HHS/MicroHealth. 
- MicroHealth and HHS will procure funding for research efforts to be held in Tremendous. 
- MicroHealth (as "Admin") will provide user accounts to Nava designers (as "Member" w/o access to billing information and settings) that can be used to distribute incentives to research participants. 

-----

**Options explored:** 
| | MH-procured Ethnio | MH-procured Tremendous + other manual tools & processes | Nava's Ethnio + MH-procured Tremendous |
| --- | --- | --- | --- |
| **Participant management** | ✅ | ❌ — Manual spreadsheet management requires additional processes to secure PII | ✅ |
| **Incentives disbursement** | ✅ | ✅  | ✅ | 
| **Scheduling** | ✅ | ❓ — Manual process (email coordination, manually created Zoom meetings)  | ✅ |
| **Screeners & Intercepts** | ✅ | ❓ — via Google Forms | ✅ |
| **Consent Forms** | ✅ | ❓ — via Google Forms | ✅ | 
| **Summary** | ❌ — Everything needed but cost-prohibitive | ❌ — Addition manual overhead & potential PII risks  | ✅


/cc @margaretspring 

Addresses #1634

**Comment by andycochran** [comment]
Resolving this. New issue created for setting up incentive tooling: https://github.com/HHS/simpler-grants-gov/issues/1701
